wsgi/odyssy/odyssy/urls.py

```diff
@@ -30,5 +30,9 @@
     url(r'^news/', include('news.urls')),
     url(r'^tag/', include('tag.urls')),
     url(r'^announcement/', include('announcement.urls')),
+    url(r'.html/$', include('serve_static.urls')),
```

Comment:
Such apps are already there for Django. Please take a look at https://docs.djangoproject.com/en/1.11/ref/contrib/staticfiles/.